db/migrate/20220114195311_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :commentscounter
+      t.integer :likescounter
```

Comment:
Let's work with ruby/rails naming conventions here also. 🙂 
```
t.integer :commentscounter
t.integer :likescounter
```
should be
```
 t.integer :comments_counter
 t.integer :likes_counter 
```

Reply:
Done. Thank you @bimbolabuari